object.go

```diff
@@ -34,9 +34,27 @@ func (c Color) MarshalText() ([]byte, error) {
 	return []byte(c), nil
 }
 
+type DatabaseMention struct {
+	ID ObjectID `json:"id"`
+}
+
+type PageMention struct {
+	ID ObjectID `json:"id"`
+}
+
+type Mention struct {
+	Type            ObjectType       `json:"type,omitempty"`
+	DatabaseMention *DatabaseMention `json:"database,omitempty"`
```

Comment:
Let's keep fields name consistent. So, `DatabaseMention` -> `Database` and `PageMention` -> `Page`

Reply:
Done